src/puppetlabs/puppetdb/command.clj

```diff
@@ -266,42 +266,45 @@
 ;; Catalog replacement
 
 (defn replace-catalog*
-  [{:keys [certname version id received payload]} db]
+  [{:keys [certname version id received payload]} start-time db]
   (let [{producer-timestamp :producer_timestamp :as catalog} payload]
     (jdbc/with-transacted-connection' db :repeatable-read
       (scf-storage/maybe-activate-node! certname producer-timestamp)
       (scf-storage/replace-catalog! catalog received))
-    (log/info (trs "[{0}-{1}] ''{2}'' command processed for {3}"
+    (log/info (trs "[{0}-{1}] [{2} ms] ''{3}'' command processed for {4}"
```

Comment:
Looks like this is formatting with thousands separators - not sure we want that (e.g. 1,2341 ms)?  It'll make parsing harder if anyone's likely to try to parse this (locale dependent).